test/solc/main.cpp

```diff
@@ -0,0 +1 @@
+
```

Comment:
The CI fails because this wasn't added for compilation. Can you remove this both locally and from the PR?